docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: 3
```

Comment:
Dont think you need this file anymore?

Reply:
Tested it, and we do.

Reply:
Ok. Can you point me to the code that triggers docker-compose? Cant find where this file is run from.

Also, this file should be in the `docker`-folder.